packages/flame/CHANGELOG.md

```diff
@@ -3,15 +3,16 @@
 ## [Next]
  - Add `ButtonComponent` backed by two `PositionComponent`s
  - Add `SpriteButtonComponent` backed by two `Sprite`s
+ - Allow more flexible construction of `EffectController`s and make them able to run back in time
  - Fix `Parallax.load` with different loading times
+ - Fix render order of components and add tests
```

Comment:
I believe this line was in section "1.0.0-releasecandidate.18" by accident, so moved it here.